src/Blocks/BlockBigFlower.h

```diff
@@ -141,6 +141,20 @@ class cBlockBigFlowerHandler:
 
 
 
+	virtual void OnPlayerBrokeBlock(
+			cChunkInterface & a_ChunkInterface, cWorldInterface & a_WorldInterface,
+			cPlayer & a_Player,
+			Vector3i a_BlockPos,
+			BLOCKTYPE a_OldBlockType, NIBBLETYPE a_OldBlockMeta
+	) const override
+	{
+		OnBroken(a_ChunkInterface, a_WorldInterface, a_BlockPos, a_OldBlockType, a_OldBlockMeta);
+	}
```

Comment:
Instead of making the two one function, you could also move this default into `cBlockHandler`. That way `OnPlayerBroke` always takes precedence over `OnBroken` but you only need to define `OnBroken` to cover both cases.

Reply:
That works too - now we have to decide what is the better way

Reply:
One consideration would be future-proofing for when we need to distinguish between more than just players/other; TNT entities, endermen, enderdragons, creepers can all break blocks too.

Reply:
Zomies also e.g. doors